crates/client/rpc/Cargo.toml

```diff
@@ -53,6 +53,7 @@ mp-block = { workspace = true }
 mp-felt = { workspace = true }
 mp-hashers = { workspace = true }
 mp-transactions = { workspace = true, features = ["client"] }
+mp-simulations = { workspace = true }
```

Comment:
default-features = true

Reply:
but that's already there at the root Cargo.toml

Reply:
what's the reason for this if its working without that? we haven't added it for other packages?

Reply:
Yep, 99% of the time it is the same code being executed.
The big changes between no-std and std are:
- use of hashbrown instead of std::collection::HashMap (same for HashSet)
- `Error` trait being implemented on error types

Reply:
so should we add it for like `mp-felt` as well?